### PR TITLE
hardening/971_document_creation_of_CKAN_structures

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #971 
* No tests are required
* Assignee @frbattid

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md [hunk 1]
```diff
@@ -354,3 +358,45 @@ In detail, this is the workflow when `NGSICKANSink` is combined with `CKANCache`
...
 
+##<a name="section4"></a>Annexes
+
+###<a name="section1"></a>Provisioning a CKAN resource for the column mode
+First of all, you'll need a CKAN organization and package/dataset before creating a resource and an associated datastore in order to persist the data.
+
+Creating an organization, let's say in [`demo.ckan.org`](http://demo.ckan.org/); the organization name is `service`, because our entity will be in that FIWARE service:
+```
+$ curl -X POST "http://demo.ckan.org/api/3/action/organization_create" -d '{"name":"service"}' -H "Authorization: xxxxxxxx"
+
```
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: Unnecessary white line.

File: doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md [hunk 2]
```diff
@@ -354,3 +358,45 @@ In detail, this is the workflow when `NGSICKANSink` is combined with `CKANCache`
 
 [Top](#top)
 
+##<a name="section4"></a>Annexes
+
+###<a name="section1"></a>Provisioning a CKAN resource for the column mode
+First of all, you'll need a CKAN organization and package/dataset before creating a resource and an associated datastore in order to persist the data.
+
+Creating an organization, let's say in [`demo.ckan.org`](http://demo.ckan.org/); the organization name is `service`, because our entity will be in that FIWARE service:
```
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: I would add: "...let's say in demo.ckan.org for the sake of demonstration, but should be a CKAN deployment of yours;..."

File: doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md (hunk 1 above)
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: Fixed in 018bf8c

File: doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md (hunk 2 above)
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: Fixed in 018bf8c

File: doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md [hunk 3]
```diff
@@ -354,3 +358,44 @@ In detail, this is the workflow when `NGSICKANSink` is combined with `CKANCache`
 
 [Top](#top)
 
+##<a name="section4"></a>Annexes
+
+###<a name="section1"></a>Provisioning a CKAN resource for the column mode
```
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: Sorry, I forgot about this: It must be referenced the CKAN API: http://docs.ckan.org/en/latest/api/

Something like "This section is built upon the assumption you are familiar with the CKAN API. If not, please have a look on [it](http://docs.ckan.org/en/latest/api/)".

Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md]: Fixed in 018bf8c (and fixed the number of subsection: is 4.1, not 1 :+1:  )

**Comment by frbattid** [comment]
LGTM

**Pull request merged**